zim.cc

```diff
@@ -1446,7 +1388,7 @@ class ZimResourceRequest: public Dictionary::DataRequest
   ~ZimResourceRequest()
   {
     isCancelled.ref();
-    hasExited.acquire();
+    //hasExited.acquire();
```

Comment:
What is the meaning of commenting out rather than removing code? Are you unsure the change is correct and useful?

Reply:
the semphore seems not neccessary.
but this can be revert

Reply:
Not necessary because of the QtWebKit => QtWebEngine switch OR never necessary? If you think it is not necessary even with QtWebKit, such a hypothesis must be very carefully verified. You'd have to do a thorough analysis, including the commit history of this code. Thread safety is difficult to get right and such code shouldn't be touched lightly.

Reply:
reasonable ,I did not verify it .
just give me a feeling will cause dead lock . when try to found issues related to #1435

anyway I'll revert it .